src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/solution/problem_setup_step.py

```diff
@@ -92,6 +94,25 @@ class ProblemSetupStep(StepModel):
 
     # Methods ---------------------------------------------------------------------------------------------------------
 
+    @transaction(
+        self=StepSpec(
```

Comment:
why do we need to clean the method assets directory? I think we have a design issue here. oSL should not use the asset folder as a working directory